WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInActivity.java

```diff
@@ -195,12 +213,16 @@ public SmartLockHelper getSmartLockHelper() {
     }
 
     private void popBackStackToSignInFragment() {
-        FragmentManager fragmentManager = getSupportFragmentManager();
-        while (fragmentManager.getBackStackEntryCount() > 1) {
-            fragmentManager.popBackStackImmediate();
-        }
 
-        getSupportFragmentManager().popBackStack();
+        // this can bee called asynchronously from Volley onError handler so we must check if state is OK
```

Comment:
I guess it's possible - that exception seems possible anytime we use `commit()` :) But I'm not sure why we can't just use `commitAllowingStateLoss()` here?